LocustScripts/commcarehq-central-registry-search-and-admit.py

```diff
@@ -0,0 +1,180 @@
+import logging
+from openpyxl import load_workbook
+import random
+import time
+
+from locust import HttpUser, SequentialTaskSet, between, task, tag, events
+from locust.exception import InterruptTaskSet
+
+from user.models import UserDetails, HQUser, AppDetails
+from common.args import file_path
+from common.utils import load_json_data
+from common.web_apps import get_app_build_info
+
+@events.init_command_line_parser.add_listener
+def _(parser):
+    parser.add_argument("--domain", help="CommCare domain", required=True, env_var="COMMCARE_DOMAIN")
+    parser.add_argument("--app-id", help="CommCare app id", required=True, env_var="COMMCARE_APP_ID")
+    parser.add_argument("--app-config", help="Configuration of CommCare app", required=True)
```

Comment:
I was about to agree, but Woody brought up a scenario today where the app config would change. He wants to run the same script on a slightly different variation of the same app where the case search title is different. This can reuse everything with a slightly different app-config file